sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ier12-integration/_tests_/e2e/ier12-integration.scheduler.e2e-spec.ts

```diff
@@ -186,6 +187,13 @@ describe(describeProcessorRootTest(QueueNames.IER12Integration), () => {
         submittedDate: referenceSubmissionDate,
       },
     );
+    // Create an original assessment to be ignored by the IER12 processing.
+    await db.studentAssessment.insert(
+      createFakeStudentAssessment({
+        application,
+        auditUser: application.student.user,
+      }),
+    );
```

Comment:
IER records are created per disbursement, hence creating an assessment with disbursement with assessment or disbursement date falling withing timeframe only can assert the outcome effectively.

Reply:
As we talked, I am removing the E2E change since the query is now using a one-to-one relation with the current assessment.